config/routes.rb

```diff
@@ -29,6 +29,6 @@
 
   get "ping", to: "status#ping", format: :json
   get "healthcheck", to: "status#status", format: :json
-  get "status", to: "status#ping", format: :json
+  get "status", to: "status#status", format: :json
   get "state_benefit_type", to: "state_benefit_type#index", format: :json
```

Comment:
while we're here, can we get rid of (or document) what these endpoints are for? I find them very confusing (as clearly you have with the renaming)

Reply:
Which do you want to get rid of?

Reply:
I presume ping and healthcheck if we're no longer using them? Or at least add a comment saying what they are for